### ISSUE TITLE
[nav] - Redesign navigation bar

### ISSUE THREAD

**Issue by kristinashu** [mozilla/teach.mozilla.org#1259]
Simply the main navigation and make it more scalable. Few things to think about:
- scalable so we can add badges (and eventually an About page)
- simplify icons and text
- how to login and show user profile (possible add profile pic to align with thimble)
- make it sticky
- possibly remove different colours for each sections
- how to show secondary pages
- rethink footer #1258

**Comment by sabrinang** [comment]
Some general IA I've been thinking about in some wireframes (https://redpen.io/upceaae7c440058962) that may inform this navigation bar redesign. Noting it here for now but could b filed as separate issue.
_______________________________________________________

**ABOUT**
- Mozilla Learning and universal web literacy *(mission)*
- Web Literacy Map *(framework)*
- Teach like Mozilla *(values)*



**CURRICULUM**
- Database



**COMMUNITY**
- Directory
- Discourse

**TRAINING / PROFESSIONAL DEVELOPMENT**
- Badges

**INITIATIVES / PROGRAMS** 

- Mozilla Clubs
 - Leadership Roles: Club Captain or Regional Coordinator

- Maker Party
  - Leadership Role: Maker Party Host

- Mozilla Hive
 - Leadership Role: Hive Member

- Mozilla Fellowships
 - Leadership Roles: Fellow (in Advocacy Open Web, Science, Open News or MDN)

- Mozilla Festival
 - Leadership Role: Space Wrangler

**TOOLS**
- Thimble
- Webmaker
- X-Ray Goggles

**Comment by kristinashu** [comment]
@sabrinang love this IA! I think it will be so much clearer for users. Any idea on how/when we'll be able to start implementing this?

**Comment by hannahkane** [comment]
I also like this a lot!

cc @chadsansing to comment on the placement of Teach Like Mozilla specifically. My sense is that, while we might have some of the current TLM content in the About section (maybe in a sub-section called "Our Values"), most of the TLM content would actually fit under Training/Professional Development.

Would like to test this IA with users before implementing.

**Comment by jgmac1106** [comment]
Off topic, but these are amazing. One of the greatest assets I have gotten
out being involved in MLN in watching designers work. Not just designers
but webby Mozillian designers. I have mentioned it before but their is a
unique type of learning occurring that places design in a dance a with
inquiry.

**Comment by chadsansing** [comment]
I dig these. 

In the future (hopefully), TLM will be both a landing resource (brief, but extensively linked) and a curriculum category, largely for PD, so it may wind up with its own space, as well as on /activities as modules tagged TLM. I don't know if that helps, but that is what I'm thinking right now.

**Comment by hannahkane** [comment]
Suggestion for terminology (based on MLN team call that's happening right now): instead of "Professional Dev / Trainings," let's say "Leadership Development" and have "Teach Like Mozilla curriculum" and "Badges" as sub-items.

**Comment by sabrinang** [comment]
Also to note, resources like working open guide http://mozillascience.github.io/leadership-training/index.html (science lab put together for fellowships and was mentioned in whistler it might go on MLN?) could potentially have a home under "Leadership Development" as well

**Comment by chadsansing** [comment]
@sabrinang, definitely. cc @zee-moz.

**Comment by LauraReynal** [comment]
cc

**Comment by sabrinang** [comment]
we have mozilla fellowships considered in the upcoming nav but there are also ideas of it having its own home on https://github.com/MozillaFoundation/plan/issues/565

**Comment by sabrinang** [comment]
Mockups of how the structure of redesign could look on desktop
Note: more content to be determined and to be added so there will be more on the menu and icons are placeholders
![mln - home](https://cloud.githubusercontent.com/assets/2492510/10143329/962fbaac-65cc-11e5-94eb-dcde97c4d346.png)
In the menu you can expand a section at a time but this is how it would look if it was all was expanded to show items:
![mln - home if all were expanded](https://cloud.githubusercontent.com/assets/2492510/10143330/973ff150-65cc-11e5-9c87-193daa4e101b.png)

**Comment by hannahkane** [comment]
Nice. Would love to user test vocabulary and structure, to see if users can intuit the content in each section.

My instinct is to demote "About MLN" and maybe even "Tools" and *maybe* even "Initiatives." Could we consider an IA that has a secondary nav for items you'd visit less frequently?

**Comment by sabrinang** [comment]
redesign of navigation may be blocked by https://github.com/mozilla/teach.mozilla.org/issues/1260#issuecomment-140234632 as we think a distinct MLN logo or wordmark brand is needed

**Comment by kristinashu** [comment]
Come across this article on horizontal vs vertical nav, something to consider and keep in mind when we get to this ticket http://www.smashingmagazine.com/2010/01/the-case-against-vertical-navigation/

**Comment by hannahkane** [comment]
Interesting article. It definitely reminded me that we should focus our IA as much as possible. Looking at my comment above, I'm still very much interested in figuring out how to keep the primary nav limited to content that's likely to change or be referenced frequently. Static content should be easy to find, but isn't likely to be referred to by users very often. One possible exception to this is Clubs content, but only for Club Captains and RCs. Perhaps the experience is slightly different for those authenticated users who will likely want to get to, say, their Club reports very easily.

**Comment by hannahkane** [comment]
@kristinashu - we've done this, haven't we?

**Comment by kristinashu** [comment]
Yup I think so, closing!
